src/app/commands/cmd_cancel.cpp

```diff
@@ -47,6 +48,8 @@ void CancelCommand::onLoadParams(const Params& params)
   std::string type = params.get("type");
   if (type == "noop") m_type = NoOp;
   else if (type == "all") m_type = All;
```

Comment:
I leave the 'else if (type == "all") m_type = All;' just for backwards compatibility in case someone has used app.command.Cancel{type="all"} in some script + if in the future we decide to change the default type of the Cancel command.